gerardo-santos/react-apprenticeship-capstone1
# Feat mini challenge 3

## Description

```
In this pull request the mini challenge 3 was completed

- The videos displayed are fetched from the youtube api.
- The video details page was added.
- Videos are able to play.
- The video details page is responsive.
-
```

## Type of Change

```
Delete options that are not relevant.
```

## How should this be tested?

```
Add a brief description of how can be tested this deliverable step by step.

- It was checked to see if the app is responsive, the search bar works, videos are obtained from the API.

```

## Any background context you want to provide?

```
N/A
```

## What is the deliverable?
Mini challenge 3

```
Number of the deliverable and the tittle of this one.

Mini challenge 3
```

## Screenshots and links


![image](https://user-images.githubusercontent.com/97237961/149168845-0650472f-b4d3-43d0-bfa2-ba119eeaa4b0.png)

![image](https://user-images.githubusercontent.com/97237961/149168864-b317b374-5321-4a82-9b1b-f82f0a3922b7.png)

![image](https://user-images.githubusercontent.com/97237961/149168897-1942188a-1874-4fe0-9429-83764404117e.png)

![image](https://user-images.githubusercontent.com/97237961/149168925-61a3782b-72fd-4971-a007-780722a87397.png)

![image](https://user-images.githubusercontent.com/97237961/149168951-97be9cf5-628d-447f-ba08-727985d3cda7.png)

![image](https://user-images.githubusercontent.com/97237961/149168976-9ee5bdfd-c27e-4c4c-802f-850784a1fffa.png)


https://condescending-goldberg-344c8e.netlify.app


## Notes

```
N/A
```

## Checklist
